### PR TITLE
Update history files

### PR DESCRIPTION
**Pull request by rodrigok**

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -3484,7 +3746,7 @@
 - Remove chatops package ([#8742](https://github.com/RocketChat/Rocket.Chat/pull/8742))
 - Removed tmeasday:crypto-md5 ([#8743](https://github.com/RocketChat/Rocket.Chat/pull/8743))
 - Update meteor package to 1.8.1 ([#8802](https://github.com/RocketChat/Rocket.Chat/pull/8802))
-- Fix typo ([#8705](https://github.com/RocketChat/Rocket.Chat/pull/8705) by [@rmetzler](https://github.com/rmetzler))
```
Comment (sampaiodiego) [inline HISTORY.md]: why some contributors like this were removed?

File: .github/history.json [hunk 2]
```diff
@@ -13094,9 +13277,9 @@
           "userLogin": "Prakharsvnit",
           "milestone": "0.64.0",
           "contributors": [
-            "Prakharsvnit",
```
Comment (sampaiodiego) [inline .github/history.json]: `Prakharsvnit` was removed as well

**Comment by sampaiodiego** [comment]
should we close this?

**Comment by rodrigok** [comment]
Closing, we need to do a update some time.

**Pull request closed without merging**